### DISCUSSION TITLE
天翼的登录接口是不是改了

### DISCUSSION THREAD

**Discussion by LeeSinGOST** [AlistGo/alist#1180] [Invalid question]
今天一打开就这样：
![image](https://user-images.githubusercontent.com/58470224/171081476-97bfcdfe-18de-4855-8e6d-1386964c1fee.png)
重新添加账号之后：
![image](https://user-images.githubusercontent.com/58470224/171081301-71bfa700-66eb-4bc9-810d-06198a9fb4d1.png)
昨天还可以用

**Comment by xhofe** [comment]
用189CloudPC

**Reply by LeeSinGOST** [reply]
👌
